keepassxc-browser/content/passkeys.js

```diff
@@ -146,9 +175,6 @@ const handleError = function(errorCode, errorMessage) {
         Object.defineProperty(window.PublicKeyCredential, 'isConditionalMediationAvailable', {
             value: isConditionalMediationAvailable,
         });
-        Object.defineProperties(window.PublicKeyCredential, 'isUserVerifyingPlatformAuthenticatorAvailable', {
```

Comment:
Was removing this intentional?

Reply:
I was having constant errors in the console that overriding this is not possible.

Reply:
What browser was that with? I don't recall seeing that on Firefox.

Reply:
There was a typo. Fixed.